imap/ctl_mboxlist.c

```diff
@@ -96,6 +96,7 @@ enum mboxop { DUMP,
               CHECKPOINT,
               UNDUMP,
               VERIFY,
+              ADD_ROOTS,
```

Comment:
This really does feel like what we need is a `$V` key or something in mailboxes.db, rather than using the `AS ` key as the upgrade indicator, and then we can confirm that roots have been added.